src/install/python/python-frameworks/python-agent-fastapi.mdx

```diff
@@ -0,0 +1,19 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using FastAPI
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command, or you can use the Python agent API.
+
+### Automatic initialization with admin script (recommended) [#wrapper-script]
```

Comment:
Remove this section.

Reply:
@hmstepanek I removed this section: "Automatic initialization with admin script." I made a few minor edits.